tests/Bremsstrahlung_TEST.cxx

```diff
@@ -357,14 +200,14 @@ TEST(Bremsstrahlung, Test_of_dEdx_Interpolant)
         auto cross = make_bremsstrahlung(particle_def, *medium, ecuts, true,
                                          config);
 
-        dEdx_new = cross->CalculatedEdx(energy) * medium->GetMassDensity();
+        dEdx_new = cross->CalculatedEdx(energy);
 
-        if (particleName == "TauMinus" && energy < 1.e5)
+        if (particleName == "TauMinus" && energy < 1e5)
```

Comment:
These issues shouldn't just be a comment in the unit tests, but real issues on github.